### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by Terbeche**
## Features added on this pull request:
- Set up the repository and tools.
- Create the 2 pages for mobile (home and about)
- Adapted the 2 pages to the desktop.
- Implement the user interactions: link, mobile menu.
- Implemented the section "Featured speakers" with dynamic HTML using javaScript.
- Created a descriptive README.
- Please find hereafter a link to my description video on loom:
https://www.loom.com/share/7c31d68c0b3f493c807ac2c5fbbb4313

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
 # E-learning-platform
```
Comment (gemmen29) [inline README.md:1]: - [x] Kindly include in the video pieces of code you are interested in as in the project requirement.
image

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
-This is a E-learning-platform website for the FIRST CAPSTONE PROJECT of the microverse program
+
+>This is a E-learning-platform website for the FIRST CAPSTONE PROJECT of the microverse program
+
+Original design idea by [Cindy Shin in Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```
Comment (gemmen29) [inline README.md]: - [x] [OPTIONAL] Kindly move this to the Acknowledgments section since it already exists.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      </div>
+
+      <div id="all-courses" class="hide-me-on-mobile">
+        <p>See all courses</p>
+      </div>
+    </section>
+
+    <section id="teachers">
+
+    </section>
```
Comment (gemmen29) [inline index.html:101]: - [x] Kindly add the background image to match the Behance design.

| Your design | Behance design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Terbeche, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Terbeche) [reply README.md:1]: Hi, I talked about the **More** and **Less** button on the home page (mobile version) as it was the interesting thing that I've built in this project

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved✔️
Hi @Terbeche , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

✔️ No Linter errors
✔️ Professional PR and README
✔️ Great job applying the required changes!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**